About.html

```diff
@@ -0,0 +1,108 @@
+            <a href=""><i class="fa fa-twitter"></i></a>
+            <a href="">French</a>
+            <a href="">My Page</a>
+            <a href="">Barcelona</a>
+        </div>
+        <img src="./img/hamburger.svg" class="menu" alt="menu">
+
+        <nav class ="Desktop-navbar">
+             
+            <a href="./index.html"><img src="./img/MWClogo.png" alt="MWC" class="mylogo" /></a>
+            
+            <ul class="desktop-menu">
+
+                <li class="about"><a class="about" href="./About.html">About</a></li>
+                <li class=""><a href="">Program</a></li>
+                <li class=""><a href="">Join</a></li>
+                <li class=""><a href="">Sponsor</a></li>
+                <li class=""><a href="">News</a></li>
+                <a href=""><button type="button">CC Campaign</button></a>
+            </ul>
+        </nav>
+
+        <div class="mobile-menu">
+            <ul>
+                <li><a href="./index.html">Home</a></li>
+                <li><a href="">Program</a></li>
+                <li><a href="">Join</a></li>
+                <li><a href="">Sponsors</a></li>
+                <li><a href="">News</a></li>
+                <div class="closeMenu"><i><img src="./img/close_mobile_menu.png" alt="cross icone"></i></div>
+            </ul>
+        </div>
+
+    </header>
+
+    <section class="about-event">
+        <div class="headline">
+             
+            <h1>Mobile World Congress  <br> MWC 2023 </h1>
+            <p class="about-mwc">From blockchain to robotics, 
+                MWC is the ultimate event focusing on the future of the connectivity ecosystem. 
+                Some of the primary exhibitors and keynote speakers this year will include the CEO of Orange Group,
+                 the Chairman and CEO of Telefónica, and the President and CEO of the Ericsson Group, among others.</p>
+            <h3>Please contact us per Email for any further questions about MWC event </h3>
+            <p class="location">mwcbarcelona@mwc.com</p> 
+        </div>
+    </section>
+    <section class="home">
+        <div class="main-home">
+            <h3>MWC EVENT LOGO</h3>
+            <hr>
+            <p>Join us for a more exciting, 
+                more jam-packed show than ever before! The connectivity industry is changing fast,
+                 and this year’s theme - Velocity - will catapult you to the forefront.</p>
+            <img class ="logoimage" src="./img/mYLOGO.png" alt="Logoimage">
+        </div>
+    </section>
+
+    <section class="pastevents">
+       <div class="past">
+          <h4>SEE THE PAST MWC EVENTS</h4> 
+          <hr>
+          <p>Take a look at the two MWC events wich took place in  <b>Sydney</b> and <b>Tokyo</b></p> 
+        </div>
+        <div class="mwc-past">
+        <div class="firstevent grid"></div>
+        <div class="secondevent grid"></div>
+       </div>
+    </section>   
+ <footer>
```

Comment:
- [ ] Kindly make have a dark background for about page.
- your design
<img width="1440" alt="Screenshot 2023-02-08 at 06 38 41" src="https://user-images.githubusercontent.com/81444338/217562077-bee88016-8a99-4db5-9062-975bd98bf808.png">


- Figma design
<img width="1155" alt="Screenshot 2023-02-08 at 06 39 36" src="https://user-images.githubusercontent.com/81444338/217561808-eba8f58b-f6cf-4dec-9698-92dcc80556fd.png">

Reply:
Hello @Kossi-stack , can you please confirm if I must have the same pictures of the design because, at the project requirement, we have the right to change pictures and text. 
thanks

Reply:
- [ ] @Hisoka37 the review was talking about the footer section, as you can see in the pictures above the footer section is not dark in your design as it is in the behance design.

Reply:
@danmainah Yes I get it, thank you so much.